src/ussdapp.js

```diff
@@ -510,17 +510,40 @@ go.app = function() {
         // state_locate_show_results
         self.add('state_locate_show_results', function(name, opts) {
             var choices = [];
-            opts.poi_results.forEach(function(poi_result) {
-                choices.push(new Choice('state_locate_details', poi_result));
+            var index = 0;
+            var poi_results = opts.poi_results;
+            poi_results.forEach(function(poi_result) {
```

Comment:
If there's no open bracket (which there could be if the extra informational data is missing) then you crop off the results:
```
> var poi_results = [[1, "This"]]
> poi_results.forEach(function(poi_result) {
...     poi_name = poi_result[1].slice(0, poi_result[1].indexOf('(')).trim();
.....     choices.push([index.toString(), poi_name]);
.....     index++;
..... });
> console.log(choices);
[ [ '0', 'Thi' ] ]
```